module/platform/Webots/src/Webots.hpp

```diff
@@ -141,6 +141,9 @@ namespace module::platform {
         std::map<std::string, CameraContext> camera_context;
         uint32_t num_cameras = 0;
 
+        /// @brief Max FSR sensor value
+        float max_fsr_value;
```

Comment:
Slap a default value on that bad boy
```suggestion
        float max_fsr_value = 0;
```

Reply:
.